model/lab_case.py

```diff
@@ -21,19 +21,19 @@ def type_of_case(self):
         
         individual_types = []
         for subject in self.subjects:
-            individual_types.append(subject.type)
-        
-        if "F" not in individual_types:
+            individual_types.append(subject.subject_type.name)
+                                                            
+        if SubjectType.child.name not in individual_types:
             return LabCaseType.invalid
```

Comment:
This function returns LabCaseType, but no return type has been specified
`def type_of_case(self) -> LabCaseType:`